app/cafe.py

```diff
@@ -0,0 +1,22 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if date.today() > visitor.get("vaccine")["expiration_date"]:
+            raise OutdatedVaccineError("All friends should be vaccinated")
+        if not visitor.get("wearing_a_mask"):
+            raise NotWearingMaskError("NotWearingMaskError")
```

Comment:
you have the same error messages, use more descriptive messages, also you can use a visitor name inside them